README.md

```diff
@@ -305,6 +306,30 @@ method takes one argument:
 
 2. `chain` must return a value of the same Chain
 
+### ChainRec
+
+A value that implements the ChainRec specification must also implement the Chain specification.
+
+1. `T.chainRec((done, next, value) => g(done(value)), i) `is equivalent to `g(i)`
```

Comment:
> type of `g` is `a => T a`

@safareli You've convinced me again that the law is correct. I don't know why my mind keep going back and forth on this :)

But if we have "equivalence" law, the first one seems redundant.